test/target_test.py

```diff
@@ -47,3 +53,193 @@ def open(self, mode):
                 return None
 
         GoodTarget()
+
+
+class BaseTargetTestCase():
```

Comment:
I already started to deprecate 2.x... :P. Seriously, thanks @berkerpeksag, I'll fix.